app/controllers/people_skills/params.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module PeopleSkills
+  class Params
+
+    def initialize(params)
+      @params = params
+    end
+
+    def skills
+      return [] if @params[:skill_id].nil? || @params[:skill_id].blank?
+
+      @params[:skill_id].select { |skill| skill.present? }
```

Comment:
No idea what you are trying to achieve here.
If it is filtering the array for nil values then just do https://apidock.com/ruby/v2_6_3/Array/compact

Reply:
Furthermore as this method doesn't returns skills but skill_ids it should probably be called the part